roboragi/Anilist.py

```diff
@@ -356,10 +356,12 @@ def getClosestManga(searchText, mangaList, isLN=False):
                 mangaList.remove(manga)
 
         for manga in mangaList:
-            title_english = manga.get('title_english', '').lower() or None
-            title_romaji = manga.get('title_romaji', '').lower() or None
-            mangaNameList.append(title_english)
-            mangaNameList.append(title_romaji)
+            title_english = manga.get('title_english')
+            title_romaji = manga.get('title_romaji')
+            mangaNameList.append(title_english.lower())\
```

Comment:
If this line got expanded out it'd be
```python
mangaNameList.append(title_english.lower()) if title_english else None
```
which, while valid Python (you aren't required to use values from function calls or expressions), is not pretty.

I think, in this case, the best way to handle this is to just use a plain ol' conditional.
```python
if title_english:
    mangaNameList.append(title_english.lower())
```

Reply:
Sounds good to me, I'll make the change when I get home.